### DISCUSSION TITLE
Logs in Independent Database.

### DISCUSSION THREAD

**Discussion by FradaL** [spatie/laravel-activitylog#1084] [Q&A]
I have a dedicated database just to save the logs, the problem is that when I get the causer, it tells me that the table does not exist, and it is logical since my users table is in the main database, is there any way to be able to tell $activity->causer to look up the user in the other table.

**Comment by Gummibeer** [comment]
Had the same issue in another project with other models. I had to define the connection to use in every model as otherwise Laravel will pass the connection of the current model to the related one.
